client/data/stats/use-has-never-published-post.ts

```diff
@@ -0,0 +1,38 @@
+import { useQuery } from 'react-query';
+import wpcom from 'calypso/lib/wp';
+import type { QueryOptions, UseQueryResult } from 'react-query';
+
+export const useHasNeverPublishedPostCacheKey = (
+	siteId: number | null,
+	includePages: boolean
+) => [ 'site-has-never-published-post', siteId, includePages ];
+
+interface Options extends QueryOptions {
+	enabled?: boolean;
```

Comment:
We should prefix this `boolean` variable with `is`:
```suggestion
	isEnabled?: boolean;
```

Reply:
I see now that `react-query` expects `enabled` ([documentation](https://react-query.tanstack.com/reference/useQuery)) so it's fine to keep it like this.